crates/sequencer/src/sequencer.rs

```diff
@@ -248,54 +251,91 @@ mod tests {
-                gas_prices: GasPrices {
-                    eth_l1_gas_price: 1,
-                    strk_l1_gas_price: 1,
-                    eth_l1_data_gas_price: 1,
-                    strk_l1_data_gas_price: 1,
-                },
-                use_kzg_da: false,
-
-                invoke_tx_max_n_steps: 4_000_000,
-                validate_max_n_steps: 4_000_000,
-                max_recursion_depth: 1_000,
+        let block_info = BlockInfo {
+            block_number: *ONE_BLOCK_NUMBER,
+            block_timestamp: *ONE_BLOCK_TIMESTAMP,
+            sequencer_address: *SEQUENCER_ADDRESS,
+            gas_prices: GasPrices {
+                eth_l1_gas_price: NonZeroU128::new(1).unwrap(),
+                strk_l1_gas_price: NonZeroU128::new(1).unwrap(),
+                eth_l1_data_gas_price: NonZeroU128::new(1).unwrap(),
+                strk_l1_data_gas_price: NonZeroU128::new(1).unwrap(),
             },
-            chain_info: ChainInfo {
-                chain_id: ChainId("KKRT".into()),
-                fee_token_addresses: FeeTokenAddresses {
-                    strk_fee_token_address: *STRK_FEE_TOKEN_ADDRESS,
-                    eth_fee_token_address: *ETH_FEE_TOKEN_ADDRESS,
-                },
+            use_kzg_da: false,
+        };
+
+        let chain_info = ChainInfo {
+            chain_id: ChainId("KKRT".into()),
+            fee_token_addresses: FeeTokenAddresses {
+                strk_fee_token_address: *STRK_FEE_TOKEN_ADDRESS,
+                eth_fee_token_address: *ETH_FEE_TOKEN_ADDRESS,
             },
-        }
-    }
+        };
 
-    /// Maps builtins and steps to a single cost unit of reference (gas).
-    fn vm_resource_fee_cost() -> Arc<HashMap<String, f64>> {
-        Arc::new(
-            [
-                (String::from("n_steps"), 1_f64),
-                ("pedersen_builtin".to_string(), 1_f64),
-                ("range_check_builtin".to_string(), 1_f64),
-                ("ecdsa_builtin".to_string(), 1_f64),
-                ("bitwise_builtin".to_string(), 1_f64),
-                ("poseidon_builtin".to_string(), 1_f64),
-                ("output_builtin".to_string(), 1_f64),
-                ("ec_op_builtin".to_string(), 1_f64),
-                ("keccak_builtin".to_string(), 1_f64),
-                ("segment_arena_builtin".to_string(), 1_f64),
-            ]
-            .into_iter()
-            .collect(),
+        let versioned_constants: VersionedConstants =
+            serde_json::from_str(include_str!("./resources/versioned_constants.json"))
+                .expect("failed to parse versioned constants");
+
+        let bouncer_config = BouncerConfig::max();
+
+        let concurrency_mode = Default::default();
+
+        BlockContext::new(
+            block_info,
+            chain_info,
+            versioned_constants,
+            bouncer_config,
+            concurrency_mode,
         )
+
+        // BlockContext {
```

Comment:
do we need to remove this?

Reply:
yes because the block context can't be built as before, we need the 

```rust
BlockContext::new(
            block_info,
            chain_info,
            versioned_constants,
            bouncer_config,
            concurrency_mode,
        )
```

pattern, just removed these lines